src/structures/ClientUser.js

```diff
@@ -16,17 +16,23 @@ class ClientUser extends Structures.get('User') {
   _patch(data) {
     super._patch(data);
 
-    /**
-     * Whether or not this account has been verified
-     * @type {boolean}
-     */
-    this.verified = data.verified;
-
-    /**
-     * If the bot's {@link ClientApplication#owner Owner} has MFA enabled on their account
-     * @type {?boolean}
-     */
-    this.mfaEnabled = typeof data.mfa_enabled === 'boolean' ? data.mfa_enabled : null;
+    if ('verified' in data) {
+      /**
+       * Whether or not this account has been verified
+       * @type {boolean}
+       */
+      this.verified = data.verified;
+    }
+
+    if ('mfa_enabled' in data) {
+      /**
+       * If the bot's {@link ClientApplication#owner Owner} has MFA enabled on their account
+       * @type {?boolean}
+       */
+      this.mfaEnabled = typeof data.mfa_enabled === 'boolean' ? data.mfa_enabled : null;
+    } else if (typeof this.mfaEnabled === 'undefined') {
+      this.mfaEnbled = null;
```

Comment:
Wouldn't it be "better" to move the initialization to the constructor?
Or is there any constellation where this line would be executed twice?

Reply:
as in set `this.mfaEnabled` to null  in the constructor, then use `data.mfa_enabled` in _patch? sure

Reply:
on second thoughts, wouldnt _patch be called then the property set to null that way? because it would call `super`, which then calls `this._patch`? (i also tested this in #bot-commands in the discord, seems to work that way)

Reply:
Oh, I see. I suppose it's fine like that then.